customer-commons/src/main/java/no/unit/nva/customer/model/PublicationTypes.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.customer.model;
+
+public enum PublicationTypes {
```

Comment:
There's likely an issue here because we have at some point a comparison with the actual manifestation of the types (the types are actually URIs in the data, this isn't apparent because of the JSON-LD wrapping of the data). 

Add e.g. ACADEMIC_ARTICLE("AcademicArticle") and return the string value as the JSON serialization. There is an easy way to make this change, contact me if you need help :)